plugins/destination/file/client/spec.go

```diff
@@ -0,0 +1,32 @@
+package client
+
+import (
+	"fmt"
+)
+
+type FormatType string
+
+const (
+	FormatTypeCSV  = "csv"
+	FormatTypeJSON = "json"
+)
+
+type Spec struct {
+	Directory string     `json:"directory,omitempty"`
+	Format    FormatType `json:"format,omitempty"`
+	// This is used for debugging purposes only
+	NoRotate bool `json:"no_rotate,omitempty"`
```

Comment:
Maybe we should call this `Rotate` and set the default to `true`? Maybe it's just me, but it takes me a bit of thinking to figure out whether `NoRotate=false` means it will rotate or not :) 

Also, but separately, I feel like `rotate` should actually default to false. If you just want a quick and dirty solution or have a small setup, you don't need the files to be rotated (which brings in sorting and merging problems). The defaults should fit these users; users that are productionizing their setups can spend a few more minutes to learn about the available options

Reply:
maybe in `local` it makes sense but Im trying to keep it consistent with all our "file" plugins and in most of then it will default to rotate. so don't know what is better - `inconsistency` or one of better default. I would vote for consistency across all plugins (of the same type)

Reply:
Fair enough, but can we still name it `rotate` with a documented default of `true`?

Reply:
golang has issues with `nil` values so it will mean I'll need to turn this into `*bool` this is why I made it `NoRotate`

Reply:
Okay, yeah I see. Not a big deal, let's leave it like it is. At least the meaning of `no_rotate: true` is pretty clear, and that's the only time you'd be adding it to config (because otherwise you can use the default)